### PR TITLE
feat: Financial submit cards

### PR DESCRIPTION
**Pull request by ncomerci**

### CODE REVIEW

File: src/back/routes/vestings.ts [hunk 1]
```diff
@@ -7,9 +7,14 @@ import { VestingService } from '../../services/VestingService'
 import { validateAddress } from '../utils/validations'
 
 export default routes((router) => {
+  router.get('/all-vestings', handleAPI(getAllVestings))
```
Comment (andyesp) [inline src/back/routes/vestings.ts:10]: Just `vestings`? We don't have `all-projects` or `all-proposals`

File: src/components/Updates/FinancialCard.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
+.ui.card.FinancialCard {
+  margin: 0;
+  width: 100%;
+  padding: 16px;
+  background-color: var(--black-700);
+  box-shadow: 0px 1px 8px 0px var(--alpha-black-200);
+}
+
+.ui.card.FinancialCard .Text {
```
Comment (andyesp) [inline src/components/Updates/FinancialCard.css:9]: Please use a classname to modify the Text component 🙏

File: src/components/Updates/FinancialCard.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
+.ui.card.FinancialCard {
```
Comment (andyesp) [inline src/components/Updates/FinancialCard.css:1]: I think you can avoid using `.ui.card` with some `!important`s. That is the way 🙏

File: src/components/Updates/FinancialCard.css [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+.FinancialCard__Value {
+  display: flex;
+  align-items: center;
+  margin: 8px 0;
+  gap: 10px;
+}
+
+.FinancialCard__Value > .Text {
```
Comment (andyesp) [inline src/components/Updates/FinancialCard.css:22]: Here too.

File: src/back/routes/vestings.ts (hunk 1 above)
Comment (ncomerci) [reply src/back/routes/vestings.ts:10]: Will be removed in the next PR

**Pull request merged**